migrations/postgres/2019-06-19-141114_themes/up.sql

```diff
@@ -0,0 +1,4 @@
+-- Your SQL goes here
+ALTER TABLE blogs ADD COLUMN custom_css TEXT;
+ALTER TABLE users ADD COLUMN preferred_theme VARCHAR;
```

Comment:
this should be a cookie setting

I really hope this is a cookie setting.

Reply:
Why should it be a cookie setting? Storing it in the database as the advantage of having it synced across devices (and also it will never expire).

Reply:
you'd have to sync it to the cookie, but that way it'd be easier & quicker to access.

Reply:
The whole user struct is already loaded for every page anyway, I'm not sure it is going to make a real difference :woman_shrugging: